main.go

```diff
@@ -5,12 +5,46 @@ import (
 	"fmt"
 	"os"
 
+	"github.com/olekukonko/tablewriter"
 	"github.com/optiopay/klar/clair"
 	"github.com/optiopay/klar/docker"
 )
 
+const (
+	ColorCritical = "\033[1;31m%s\033[0m"
+	ColorHigh     = "\033[0;31m%s\033[0m"
+	ColorMedium   = "\033[0;33m%s\033[0m"
+	ColorLow      = "\033[0;94m%s\033[0m"
+	ColorDefault  = "\033[0;97m%s\033[0m"
+)
+
 var store = make(map[string][]*clair.Vulnerability)
 
+var Severity = map[string]int{
+	"Defcon1":    7,
```

Comment:
Why not to map severity directly to colors?

Reply:
Sure, will do